### PR TITLE
Update README.md Pattern for Terraform plan files

### PR DESCRIPTION
**Pull request by alaamub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,34 @@ jobs:
...
+You can set Pattern for Terraform plan files (e.g., plan*). Default Value: plan*
+```
+# Trigger conditions for running this action
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+permissions:
```
Comment (drbild) [inline README.md:75]: I believe these permissions are required.  We should probably remove them from all of the examples.

**Review by drbild — APPROVED** [review]
One suggestion, but LGTM.

File: README.md (hunk 1 above)
Comment (alaamub) [reply README.md:75]: Do you mean `not required`?

Comment (drbild) [reply README.md:75]: Whoops, yep, meant not required.

**Pull request merged**